src/Normalizer/EntityHelperTrait.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Drupal\fb_instant_articles\Normalizer;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Entity\EntityChangedInterface;
+use Drupal\user\EntityOwnerInterface;
+
+/**
+ * Common entity data getters shared between the normalizers.
+ */
+trait EntityHelperTrait {
+
+  /**
+   * Instant articles config.
+   *
+   * @var \Drupal\Core\Config\ImmutableConfig
+   */
+  protected $config;
```

Comment:
Since it's a trait, probably we want to have `config()` so that if the parent class doesn't have it set, we can fallback?
```php
protected function config() {
  if (!$this->config) {
    $this->config = \Drupal::config('fb_instant_articles.settings');
  }
return $this->config;
}
```

Reply:
Ooo, I like that. Will do.